src/WordPress/Proxy.php

```diff
@@ -102,7 +102,7 @@ public function createRequest()
      */
     public function getJSONBody()
     {
-        return file_get_contents('php://input');
+        return $GLOBALS[Hooks::CLOUDFLARE_JSON];
```

Comment:
You need `getJSONBody()` for [tests](https://github.com/cloudflare/Cloudflare-WordPress/blob/56dda99cdd915521a9b1982cd7c94dbaad4df069/src/Test/WordPress/ProxyTest.php#L44).  How would you refactor?